### DISCUSSION TITLE
automorphism (groups)

### DISCUSSION THREAD

**Discussion by micjoswig** [oscar-system/Oscar.jl#1563] [Ideas]
(1) Currently, there are functions named "automorphisms" and "automorphism_group".  To me it seems that the return types and naming convention are not uniform.

Please check.

(2) We need automorphism groups of polytopes, cones, simplicial complexes etc from polymake.

Add an additional parameter to pick the domain where that group operates by permutations.

**Comment by fieker** [comment]
some detail:
```
g = Graphs.complete_graph(4)
automorphism_group(g)
automorphisms(g)
Graphs.automorphisms(g)
```
Comment: 
 - complete_graph is not exported, is this be design?
 - automorphism_group does not exist
 - automorphisms is not attached to "the" automorphisms, it is Graphs.automorphsms
 - the autmorphisms are none... They should either be (I think)
   - maps: g -> g
   - actual permutations
   with a preference of the first

**Comment by fieker** [comment]
This is on top of @micjoswig s suggestion to rename, globally, automorphisms -> automorphisms_list

**Comment by micjoswig** [comment]
Let's briefly talk about this later today at the usual gather.  Probably the easiest would be to sit together right afterwards and do a few lines of coding together, online.

**Reply by micjoswig** [reply]
@lkastner @thofma

**Comment by fieker** [comment]
see #1565
